### PR TITLE
Create and test selector-no-combinator

### PR DESCRIPTION
**Pull request by m-allanson**

### CODE REVIEW

File: src/rules/selector-no-combinator/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# selector-no-combinator
+
+Disallow combinators.
```
Comment (jeddy3) [inline src/rules/selector-no-combinator/README.md]: This should be "Disallow combinators in selectors" to match the description conventions.

**Comment by jeddy3** [comment]
Sweet! This one also looks great. One tiny docs change is all.

**Comment by davidtheclark** [comment]
Also good. Thanks.

**Comment by jeddy3** [comment]
Odd. I can't seem to merge. Is a rebase required?

**Comment by davidtheclark** [comment]
Yep, that's usually the case.

**Comment by m-allanson** [comment]
Ah it's probably `src/rules/index.js`, I'll rebase my branch.

**Comment by jeddy3** [comment]
Thanks for the rebase. Gone in!

**Comment by m-allanson** [comment]
Thanks!

**Pull request merged**